common/git.go

```diff
@@ -112,17 +114,22 @@ func findGitSlug(url string) (string, string, error) {
 }
 
 func findGitDirectory(fromFile string) (string, error) {
-	log.Debugf("Searching for git directory in %s", fromFile)
-	fi, err := os.Stat(fromFile)
+	absPath, err := filepath.Abs(fromFile)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Should return err here, not print it...otherwise code will continue with an empty absPath